src/main/java/be/syntra/devshop/DevshopFront/models/dtos/ProductListAndMinMaxPrice.java

```diff
@@ -0,0 +1,19 @@
+package be.syntra.devshop.DevshopFront.models.dtos;
+
+import be.syntra.devshop.DevshopFront.models.Product;
+import lombok.*;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+@Builder
+public class ProductListAndMinMaxPrice {
```

Comment:
this is an exact copy of our ProductList, but with extra fields. Seems to me, you could do the same as you did in the BE. Just add the extra fields to ProductList and remove this class?

Reply:
Added properties to ProductList and replaced the ProductListAndMinMaxPrice class